### ISSUE TITLE
When opening a new tab, Wilder's menu list does not appear

### ISSUE THREAD

**Issue by Flo-Slv** [gelguy/wilder.nvim#125]
Hi there !

First, thanks for this plugin, I really appreciate it !

I just found a "bug". Maybe I missed something ?

When I opening a new tab, it's not display anymore Wilder's menu list when I type ":", "/" or "?".

I am working with Vim 8.2.

Here is my plugin list (with vim-plug).

```sh
" Plugins
call plug#begin()

Plug 'drewtempelmeyer/palenight.vim'
Plug 'preservim/nerdtree'
Plug 'ryanoasis/vim-devicons'
Plug 'tiagofumo/vim-nerdtree-syntax-highlight'
Plug 'Xuyuanp/nerdtree-git-plugin'
Plug 'vim-airline/vim-airline'
Plug 'vim-airline/vim-airline-themes'
Plug 'tpope/vim-fugitive'
Plug 'tpope/vim-commentary'
Plug 'tpope/vim-surround'
Plug 'gcmt/taboo.vim'
Plug 'mattn/emmet-vim'
Plug 'AndrewRadev/tagalong.vim'
Plug 'ap/vim-css-color'
Plug 'junegunn/vim-slash'
Plug 'junegunn/fzf', { 'do': { -> fzf#install() } }
Plug 'junegunn/fzf.vim'
Plug 'mhinz/vim-startify'
Plug 'gelguy/wilder.nvim'
Plug 'psliwka/vim-smoothie'
Plug 'kshenoy/vim-signature'

Plug 'valloric/youcompleteme'
" cd ~/.vim/plugged/youcompleteme
" python3 install.py --all

call plug#end()
```

Here is my Wilder configuration in .vimrc

```sh
" WILDER
autocmd VimEnter * ++once call s:wilder_init()

function! s:wilder_init() abort
        call wilder#setup({'modes': [':', '/', '?']})

        call wilder#set_option('pipeline', [
                \   wilder#branch(
                \     wilder#cmdline_pipeline({
                \       'language': 'vim',
                \       'fuzzy': 1,
                \       'fuzzy_filter': wilder#vim_fuzzy_filter(),
                \     }),
                \     wilder#search_pipeline(),
                \   ),
        \ ])

        call wilder#set_option('renderer', wilder#popupmenu_renderer(wilder#popupmenu_border_theme({
                \ 'highlighter': wilder#basic_highlighter(),
                \ 'min_width': '40%',
                \ 'max_height': '20%',
                \ 'reverse': 0,
                \ 'left': [
                        \   ' ', wilder#popupmenu_devicons(),
                \ ],
                \ 'right': [
                        \   ' ', wilder#popupmenu_scrollbar(),
                \ ],
        \ })))
endfunction
```

Any ideas or suggestions ?

Thanks Flo

**Comment by gelguy** [comment]
Thanks for the trying out the plugin and for the reporting the bug!

I've pushed a fix for this in b59648ad8588bcba377f4eecdea317796ebd1f9d. Could you pull the latest `master` and check if the bug is fixed for you?

**Comment by Flo-Slv** [comment]
Thanks, it's working like a charm ! :+1:
